### DISCUSSION TITLE
Using "custom" LaTex commands generates extra spacing in bullet lists

### DISCUSSION THREAD

**Discussion by tvatter** [quarto-dev/quarto-cli#6964] [Q&A]
I'm trying to use custom LaTeX commands defined via `\newcommand` in bullet lists (e.g., custom math symbols or colors). However, when using such commands in Quarto, extra spacing is added when rendering the PDF. See below a `MWE.qmd`, where the included LaTeX header only contains `\newcommand\donothing[1]{#1}`:

```
---
title: "MWE"
format: 
    pdf:
        include-in-header: header.tex
---

No extra spacing:

* foo
* bar

Using `\newcommand\donothing[1]{#1}` creates extra spacing:

* \donothing{foo}
* bar

Adding text after `\donothing` suppresses the extra spacing:

* \donothing{foo} LOL
* bar

This stays true when the text following the extra spacing is commented out:

* \donothing{foo} <!-- LOL -->
* bar
```

![image](https://github.com/quarto-dev/quarto/assets/5273142/467c1690-bf43-4a49-a2d1-81d1dc2e36a5)

What am I doing wrong? Is there an easy solution that I'm missing?

**Comment by cscheid** [comment]
You can try `` `\donothing{foo}`{=latex} ``, which is Pandoc's syntax for a RawInline node. You might also find the `keep-tex: true` YAML option useful for debugging intermediate .tex code.

**Reply by remlapmot** [reply]
Using `` `\donothing{foo}`{=latex} `` seems to be a good fix.

For a reason I cannot explain, the original code causes Pandoc to omit a `\tightlist` from the itemize environment, which causes the extra spacing. Whereas with this, and the other methods found, the itemize environment is given a `\tightlist`. To see this, rendering the original version with `keep-tex: true`, the `.tex` file shows

```latex
\begin{itemize}
\item
  \donothing{foo}
\item
  bar
\end{itemize}
```

Whereas, rendering with `keep-tex: true` and using `` `\donothing{foo}`{=latex} ``, the `.tex` file shows

```latex
\begin{itemize}
\tightlist
\item
  \donothing{foo}
\item
  bar
\end{itemize}
```
